### ISSUE TITLE
LaunchPad - Set up email reminders.

### ISSUE THREAD

**Issue by Addison-Stavlo** [Automattic/wp-calypso#70071]
Some slack context in this thread: p1667593156541869/1667518943.518509-slack-CHN6J22MP

We should ensure email reminders are being sent to users who have visited the launchpad and not yet completed it. Designs and some context in the linked slack thread above.

We should also look into setting up an simple way to determine which LaunchPad step they left off at, in which case the reminder can be more specific to where in the LaunchPad they left off.

**Comment by agrullon95** [comment]
User should now be receiving emails for the Newsletter and Link-in-bio.

Email previews:

1st email reminder: https://mc.a8c.com/email-previewer/?theme=wpcom-2022&template=onboarding-use-case-launchpad-start-1&locale=en&variables%5Bcta_url%5D=https%3A%2F%2Fwordpress.com%2F&variables%5Bcta_label%5D=Click+here

2nd email reminder: https://mc.a8c.com/email-previewer/?theme=wpcom-2022&template=onboarding-use-case-launchpad-start-2&locale=en&variables%5Bcta_url%5D=https%3A%2F%2Fwordpress.com%2F&variables%5Bcta_label%5D=Click+here

Email after the site has been launched: https://mc.a8c.com/email-previewer/?theme=wpcom-2022&template=onboarding-use-case-launchpad-success-1&locale=en&variables%5Bcta_url%5D=https%3A%2F%2Fwordpress.com%2F&variables%5Bcta_label%5D=Click+here

**Comment by agrullon95** [comment]
LaunchPad Newsletter and Link-in-bio email reminders have been set up by Dinobots team: pd0D4C-W2-p2
I have tested them out and I was able to receive the three emails set up for the LaunchPad flows.

First email reminder:  Email sent out to users after they land on the LaunchPad screen.
Second email reminder: Email sent out to users after one day if they haven't launched their site.
Third email reminder: Email sent out to users after they have successfully launched their site.

> We should also look into setting up an simple way to determine which LaunchPad step they left off at, in which case the reminder can be more specific to where in the LaunchPad they left off.

We still have room for improvement/brainstorming for sending email reminders that are more specific to a step that a user may have left off at. One idea is adding another site option that we can update with the step that was last selected. That could at least give us some information about the step a user left off at. The Guides framework can make use of this value to send out an email reminder with different content based on that value (we would need new templates or new content for the existing templates).
